### ISSUE TITLE
error TS2305: Module '"…/winston/index"' has no exported member 'TransportInstance' | 'TransportOptions'.

### ISSUE THREAD

**Issue by KeithGillette** [winstonjs/winston-mongodb#133]
When attempting to transpile a `typescript@2.9.2` project importing `winston@3.0.0` & `winston-mongodb@4.0.3, we get the following TypeScript errors:

```
node_modules/winston-mongodb/lib/winston-mongodb.d.ts(6,10): error TS2305: Module '"/Users/keith.gillette/Code/TaskTrain/node_modules/winston/index"' has no exported member 'TransportInstance'.
node_modules/winston-mongodb/lib/winston-mongodb.d.ts(6,41): error TS2305: Module '"/Users/keith.gillette/Code/TaskTrain/node_modules/winston/index"' has no exported member 'TransportOptions'.
```

**Comment by sourabh8003** [comment]
#140 Fixed typescript errors.

**Comment by xduseko** [comment]
Is this going to be fixed? Can't use mongo transport with typescript :-(

**Comment by yurijmikhalevich** [comment]
Fix merged, released as v4.0.4.
